app/wasm/bindings/pool.go

```diff
@@ -0,0 +1,8 @@
+package wasmbindings
+
+import sdk "github.com/cosmos/cosmos-sdk/types"
+
+type PoolAssets struct {
+	Assets []sdk.Coin
+	Shares sdk.Coin
+}
```

Comment:
As part of the refactor, we're trying to convert a lot of instances of `PoolAssets` in the code / other modules, to be PoolLiquidity. We should probably revisit this wasm binding pre-release

Reply:
We can gladly rename everything. Changing the type I easy and this is only used when querying the gamm, so please do suggest new names.

We use the [Rust definitions as the canonical source](https://github.com/confio/osmosis-bindings/blob/main/packages/bindings/src/query.rs#L68-L74) for all types that hit the contract boundary, and this is more permanent (breaking API changes), so maybe you can double-check we use the proper names there first. Types that are only used within this package can easily be renamed by anyone

Reply:
Awesome! Will review those in more depth then